dch/DB.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require('mongoose'); //include mongose moduel 
+const Schema = mongoose.Schema;
+const mongoDB_url = 'mongodb://localhost:27017/accelration_user_data';
+
+mongoose.connect(mongoDB_url); 
+//Sucessfully connected to the databse
+mongoose.connection.on('connected', function() {
+	console.log('Connected to database');
+});
+
+//Failed the connection to the database
+mongoose.connection.on('err', function(err) {
+	console.log('Unable to connect to the databse '+ err);
+
+});
+
+mongoose.connection.on('disconnect', function(err) {
+	console.log('The database has been disconnected');
+});
+
+const accelrationDataSchema = new Schema({
+	watch_id: String, 
+	acc_x: Number, 
+	acc_y: Number, 
+	acc_z: Number, 
+	timestamp: Number
+	});
+
+module.exports.disconnect = function(){
+		mongoose.disconnect();
+};
+
+module.exports.getData = function(watch_id, startTime, stopTime, freq){
+//TODO: return string for gettingd data throw an error if there is a problem
+	var returnData = JSON.stringify("{watchId: ['watchi1', 'watch id 2', 'watch id 3']}"); 
```

Comment:
Did you test this? ``` JSON.stringify(); ``` takes an object literal as a param. Not a string. Also, here 'watchId' in camelcase actually breaks the schema (I know, I gave you a comment on the parameter but in this case, 'watch_id' must follow the same format if it is to be recognized properly by code that connects to MongoDB). If you just want to return an object literal, there is no need to parse a string and create it.

If you need the 'watchId' parameter in the object Why not write:

```
var returnData = {
  watch_id: ['watchi1', 'watch id 2', 'watch id 3']
};
```